src/Phansible/Resources/views/bundles/languages/php.html.twig

```diff
@@ -50,5 +50,21 @@
                 {% endfor %}
             </select>
         </div>
+        <div class="field">
```

Comment:
Can we add a link to https://blackfire.io/account/credentials or something? I think it would be handful